src/main/docker/duckdb/Dockerfile

```
@@ -1,8 +1,12 @@
-FROM ubuntu:24.10
+# DuckDB CLI container image.
+# Co-authored by: OpenCode and benizzio
+FROM ubuntu:24.04
 
 WORKDIR /duckdb
 
-RUN apt update -y && apt upgrade -y && apt install -y wget unzip && apt clean
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)